SPECS/http-nbd-transfer.spec

```diff
@@ -36,6 +36,9 @@ PYTHON=%{__python3} %{__python3} ./setup.py install --single-version-externally-
 %{_libdir}/nbdkit/plugins/nbdkit-multi-http-plugin.so
 
 %changelog
+* Thu Jul 10 2025 Mathieu Labourier <mathieu.labourier@vates.tech> - 1.7.0-1
+- Fix missing regex import
```

Comment:
Explaining what this changes would be better than this line which will only speak to the developers of this component, and even they would have to go check what the issue was if they didn't already know.

Reply:
How about "Prevent logging errors caused by missing regex import"?

Reply:
Or 

```
- Fix missing import exceptions in log files.
- Fix a potential HA startup failure with LINSTOR.
```

Reply:
done